doc/radosgw/multisite.rst

```diff
@@ -391,34 +403,37 @@ Now we update the period and commit the changes::
     }
 
 
-Starting the radosgw
---------------------
+Starting the Ceph Object Gateway instance
+-----------------------------------------
 
-Before starting the radosgw, the rgw zone and port options need to be mentioned
-in the configuration file. For more details refer to the `Install Ceph Gateway`_
-section of the guide. The configuration section for radosgw should resemble::
+Before starting the Ceph Object Gateway instance, the ``rgw_zone`` and
+``rgw_frontends`` options need to be mentioned in the configuration
+file. For more details refer to the `Install Ceph Gateway`_ section of
+the guide. The configuration section for Ceph Object Gateway instance
+should resemble::
 
   [client.rgw.us-east-1]
   rgw_frontends="civetweb port=80"
-  rgw_zone=us-east
+  rgw_zone=us-east-1
 
-And start the Ceph Object gateway (according to the OS installation) ::
+And start the Ceph Object Gateway instance(according to the OS
+installation) ::
 
-  sudo systemctl start ceph-radosgw.service
+  $ sudo systemctl start ceph-radosgw.service
```

Comment:
Here the correct command would be ceph-radosgw@rgw.us-east-1 (according to the conf)